cmd/meroxa/turbine/javascript/version.go

```diff
@@ -3,23 +3,27 @@ package turbinejs
 import (
 	"context"
 	"fmt"
+	"strings"
 
-	utils "github.com/meroxa/cli/cmd/meroxa/turbine"
+	"github.com/meroxa/cli/cmd/meroxa/turbine/javascript/internal"
 )
 
-// GetVersion calls turbine-js-cli to get the turbine-js-framework version used by the given app.
 func (t *turbineJsCLI) GetVersion(ctx context.Context) (string, error) {
-	stdOut, err := RunTurbineJS(ctx, t.logger, "version", t.appPath)
-	fmtErr := fmt.Errorf(
-		"unable to determine the version of turbine-js-framework used by the Meroxa Application at %s: %s",
-		t.appPath, err)
-	if err != nil {
-		return "", fmtErr
-	}
+	cmd := internal.NewTurbineCmd(
+		ctx,
+		t.appPath,
+		internal.TurbineCommandVersion,
+		map[string]string{},
+	)
+	cmd.Stderr = nil
```

Comment:
this is not necessary, since you are using `CombinedOutput` which will replace the stderr/out with a buffer writer.

see https://cs.opensource.google/go/go/+/refs/tags/go1.20.4:src/os/exec/exec.go;l=1002-1004

Reply:
actually internal command sets it to the defaults. blah

maybe add this to internal from ruby to javascript https://github.com/meroxa/cli/blob/master/cmd/meroxa/turbine/ruby/internal/cmd.go#LL21C1-L27C2